### PR TITLE
DEVDOCS-6108 [new]: GraphQL Storefront API, add multi-language support

### PR DESCRIPTION
**Pull request by bc-andreadao**
# [DEVDOCS-6108]

## What changed?
* Add new doc for multi-language support 

## Release notes draft
- Add [multi-language support](https://developer.bigcommerce.com/docs/storefront/graphql/multi-language-support) for a single channel. You can now query product information for a locale using the [Storefront GraphQL API](https://developer.bigcommerce.com/docs/storefront/graphql).

## Anything else?
cc @bigcommerce/team-storefront 

[DEVDOCS-6108]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-6108?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/storefront/graphql/multi-language-support.mdx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+The response includes the following:
+- **Enabled locales:** The list of the enabled locales for a channel. Includes the default locale and the shopper languages in the browser cookies.
+  <br />
+- **Resolved locale:** Indicates the locale that was selected to contain the returned data.
+
+  If you chose a default locale in the control panel, the resolved locale is the default locale.
+
+  If you chose to override the default locale with the shopper browser language, the API uses the browser's cookies to determine a shopper's language preference. To override shopper's preference, you can specify a preferred locale through the `Accept-Language` header or a GraphQL directive when you send the query. The directive takes precedence over the `Accept-Language` header<br />
+  <br />
+- **Product data for the resolved locale:** If there is no product data for the resolved locale, the response defaults to global product data.
```
Comment (bc-andreadao) [inline docs/storefront/graphql/multi-language-support.mdx:28]: Not sure if this is accurate, if someone could confirm

Comment (bc-tayvs) [reply docs/storefront/graphql/multi-language-support.mdx:28]: @jyuricle @andrii-lenets Can you confirm the statement above?

**Review by bc-tayvs — APPROVED** [review]

**Review by kskomro — DISMISSED** [review]

File: docs/storefront/graphql/multi-language-support.mdx (hunk 1 above)
Comment (andrii-lenets) [reply docs/storefront/graphql/multi-language-support.mdx:28]: @bc-andreadao yes, that's correct. The response will fallback to the global values if overrides are unavailable.

Comment (bc-andreadao) [reply docs/storefront/graphql/multi-language-support.mdx:28]: Great, thanks

File: docs/storefront/graphql/multi-language-support.mdx [hunk 2]
```diff
@@ -0,0 +1,114 @@
+# Multi-Language Support in GraphQL Storefront API
+
+The Storefront GraphQL API supports queries for multiple languages translations within a storefront channel.
```
Comment (bc-traciporter) [inline docs/storefront/graphql/multi-language-support.mdx:3]:
```suggestion
The Storefront GraphQL API supports queries for multiple language translations within a storefront channel. 
```

File: docs/storefront/graphql/multi-language-support.mdx [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+## Prerequisites
+
+- **Create language translations:**
+
+  Set the product data for a locale using the GraphQL Admin API. For example queries, see the [International Enhancements for Multi-Storefront](/docs/store-operations/catalog/msf-international-enhancements) overview.
+  <br />
+- **Choose a default locale:**
+
+  You can choose a default locale for the storefront channel, default to the global locale, or choose to override the default locale with the shopper browser language. Read more about how to [choose a default locale](https://support.bigcommerce.com/s/article/International-Enhancements-for-Multi-Storefront#localization) through the control panel. These locale settings affect which locale the API resolves.
```
Comment (bc-traciporter) [inline docs/storefront/graphql/multi-language-support.mdx:15]:
```suggestion
  You can choose a default locale for the storefront channel, default to the global locale, or override the default locale with the shopper browser language. Read more about how to [choose a default locale](https://support.bigcommerce.com/s/article/International-Enhancements-for-Multi-Storefront#localization) through the control panel. These locale settings affect which locale the API resolves.
```

File: docs/storefront/graphql/multi-language-support.mdx [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+## Returned data
+
+The response includes the following:
+- **Enabled locales:** The list of the enabled locales for a channel. Includes the default locale and the shopper languages in the browser cookies.
+  <br />
```
Comment (bc-traciporter) [inline docs/storefront/graphql/multi-language-support.mdx:21]:
```suggestion
not sure how this <br /> will display on the site. 
```

File: docs/storefront/graphql/multi-language-support.mdx [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+- **Resolved locale:** Indicates the locale that was selected to contain the returned data.
+
+  If you chose a default locale in the control panel, the resolved locale is the default locale.
+
+  If you chose to override the default locale with the shopper browser language, the API uses the browser's cookies to determine a shopper's language preference. To override shopper's preference, you can specify a preferred locale through the `Accept-Language` header or a GraphQL directive when you send the query. The directive takes precedence over the `Accept-Language` header<br />
```
Comment (bc-traciporter) [inline docs/storefront/graphql/multi-language-support.mdx:26]:
```suggestion
  If you chose to override the default locale with the shopper browser language, the API uses the browser's cookies to determine a shopper's language preference. To override a shopper's preference, specify a preferred locale through the `Accept-Language` header or a GraphQL directive when you send the query. The directive takes precedence over the `Accept-Language` header<br />
```

Comment (bc-traciporter) [reply docs/storefront/graphql/multi-language-support.mdx:26]: not sure how the <br will render

**Review by bc-traciporter — COMMENTED** [review]
I had just small changes that you can take or leave. Did you check to see how the br tag would render in the docs?

**Comment by bc-andreadao** [comment]
Thanks @bc-traciporter , yes, I checked the <br /> tags in the preview

**Pull request merged**